### PR TITLE
Move TCL test unit/type/sint to Go case

### LINKED ISSUE #822: Move TCL test unit/type/sint to Go case

### PR DESCRIPTION
**Pull request by tanruixiang**
It closes #822

### CODE REVIEW

File: tests/gocase/unit/type/sint/sint_test.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+		require.EqualValues(t, []interface{}{"1", "2", "3", "4"}, rdb.Do(ctx, "SIRANGEBYVALUE", "mysi", 1, "(5").Val())
+
+		require.EqualValues(t, []interface{}{"5", "4", "3", "2"}, rdb.Do(ctx, "SIREVRANGEBYVALUE", "mysi", 5, "(1").Val())
+
+		require.EqualValues(t, []interface{}{int64(1), int64(0), int64(1)}, rdb.Do(ctx, "SIEXISTS", "mysi", 1, 88, 2).Val())
+
+		require.EqualValues(t, 1, rdb.Do(ctx, "SIREM", "mysi", 2).Val())
+
+	})
```
Comment (tisonkun) [inline tests/gocase/unit/type/sint/sint_test.go]: I'd prefer to remove the redundant empty lines.

**Review by tisonkun — DISMISSED** [review]
LGTM. Thanks for your contribution!

Minor comments inline.

File: tests/gocase/unit/type/sint/sint_test.go (hunk 1 above)
Comment (tanruixiang) [reply tests/gocase/unit/type/sint/sint_test.go]: > I'd prefer to remove the redundant empty lines.

Ok. Done it.

**Review by tisonkun — APPROVED** [review]

**Comment by tisonkun** [comment]
Thank you! Merging...

**Pull request merged**